coldfront/plugins/qumulo/static/allocation.js

```diff
@@ -22,30 +22,43 @@ let confirmed = false;
 
 const submitButton = document.getElementById("allocation_form_submit");
 submitButton.addEventListener("click", (event) => {
-  const id_project_pk_elem = document.getElementById("div_id_project_pk");
-  const smb = protocols.find((protocol) => protocol.value === "smb");
+  const smb = protocols.find((protocol) => protocol.value === "smb"),
+        modalShouldDisplay = () => {
```

Comment:
While technically valid, I would request separating declarations of a basic variable type and the method.  I.E. give `modalShouldDisplay` its own `const` on it's own distinct statement